src/components/EventList.js

```diff
@@ -59,7 +59,7 @@ export const EventList = ({
   return (
     <SectionList
       initialNumToRender={
-        data?.length < MAX_INITIAL_NUM_TO_RENDER ? data.length : MAX_INITIAL_NUM_TO_RENDER
+        data?.length < MAX_INITIAL_NUM_TO_RENDER ? data.length + 1 : MAX_INITIAL_NUM_TO_RENDER
```

Comment:
do we need to add +1 always, also for `MAX_INITIAL_NUM_TO_RENDER` in order to render 20 instead of 19 maybe?

Reply:
Good point. I'm not sure that's necessary. I think pagination comes into play when you get to the end of the list anyway. 👍

Reply:
yes, at the end, when we load more available entries to lists

Reply:
So I don't think we need `MAX_INITIAL_NUM_TO_RENDER` + 1.

Reply:
but if we want to render always 20 and this does not work we maybe miss some entries?

Reply:
I have tried it for more than 20 event days (28.06) without any problems.

Reply:
what if we use always `initialNumToRender={MAX_INITIAL_NUM_TO_RENDER}`? if we have just 5 events, it is also ok to have initial number to 20, right?

Reply:
I don't think it'll be a problem, but I'd still prefer to keep it like this. I think it could cause errors that we can't see at the moment.

Reply:
but what is the benefit of having a more complex logic with some random +1?
if we have 5 data, it should not matter if we have initial num to render 5,6 or 20.
if we have 25 data, it should be 20.
so having 20 as initalnumtorender should be fine all the time without confusing conditions or logics.
you know what i mean?

Reply:
Yeah, I understand. I think you're absolutely right. It shouldn't matter 5 or 6 or 30. Anyway, if the size of the data is more than 20, 20 cells are rendered. If there are 5 data, there will be 5.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/855/commits/1588ac1d17b0e4914c02cd9ffe7acbe0bae91239